app/controllers/devise_token_auth/concerns/resource_finder.rb

```diff
@@ -20,7 +20,9 @@ def get_case_insensitive_field_from_resource_params(field)
   end
 
   def find_resource(field, value)
-    @resource = if resource_class.try(:connection_config).try(:[], :adapter).try(:include?, 'mysql')
+    adapter = resource_class.try(:connection_db_config)&.configuration_hash&.try(:[], :adapter)
```

Comment:
https://api.rubyonrails.org/classes/ActiveRecord/ConnectionHandling.html#method-i-connection_config

Since it's no longer a hash but an object, wouldn't it be enough to transform that line into something like

`adapter = resource_class.try(:connection_db_config)&.try(:adapter)`